ADDONS_SUPPORT.md

```diff
@@ -1,23 +1,23 @@
 ## Addon / Framework Support Table
 
-| | [React](app/react)|[React Native](app/react-native)|[Vue](app/vue)|[Angular](app/angular)| [Polymer](app/polymer)| [Mithril](app/mithril)| [HTML](app/html)| [Marko](app/marko)| [Svelte](app/svelte)| [Riot](app/riot)| [Ember](app/ember)|
-| ----------- |:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|:-------:|
-|[a11y](addons/a11y)                         |+| |+|+|+|+|+|+| | |+|
-|[actions](addons/actions)                   |+|+|+|+|+|+|+|+|+|+|+|
-|[backgrounds](addons/backgrounds)           |+|*|+|+|+|+|+|+|+|+|+|
-|[centered](addons/centered)                 |+| |+|+| |+|+| |+| |+|
-|[events](addons/events)                     |+| |+|+|+|+|+|+| | |+|
-|[graphql](addons/graphql)                   |+| | | | | | | | | | |
-|[google-analytics](addons/google-analytics) |+|+|+|+|+|+|+|+|+|+|+|
-|[info](addons/info)                         |+| | | | | | | | | | |
-|[jest](addons/jest)                         |+| | |+| | |+| | | | |
-|[knobs](addons/knobs)                       |+|+*|+|+|+|+|+|+|+|+|+|
-|[links](addons/links)                       |+|+|+|+|+|+|+| |+|+|+|
-|[notes](addons/notes)                       |+|+*|+|+|+|+|+| |+|+|+|
-|[options](addons/options)                   |+|+|+|+|+|+|+| |+|+|+|
-|[cssresources](addons/cssresources)         |+| |+|+|+|+|+|+|+|+|+|
-|[storyshots](addons/storyshots)             |+|+|+|+| | |+| |+|+| |
-|[storysource](addons/storysource)           |+| |+|+|+|+|+|+|+|+|+|
-|[viewport](addons/viewport)                 |+| |+|+|+|+|+|+|+|+|+|
+| | [Angular](app/angular)|[Ember](app/ember)|[HTML](app/html)|[Marko](app/marko)|[Mithril](app/mithril)|[Polymer](app/polymer)|[Preact](app/preact)|[React](app/react)|[React Native](app/react-native)|[Riot](app/riot)|[Svelte](app/svelte)|[Vue](app/vue)|
```

Comment:
It's alphabetical =) I mean chronological, is where preact that have been added last will be in the end =)

Reply:
🤦